DESCRIPTION

```diff
@@ -31,4 +31,6 @@ Suggests:
     vdiffr,
     testthat (>= 3.0.0)
 Config/testthat/edition: 3
+Remotes:
```

Comment:
Is plan to move back to CRAN version of EpiNow2 in future?

Reply:
Yes, this is a temporary fix while `EpiNow2` is archived from CRAN, there is an issue set as a reminder to change it back once fixed.